cranelift/filetests/filetests/isa/x64/stackslot.clif

```diff
@@ -0,0 +1,50 @@
+test compile precise-output
+target x86_64
+
+function %f0() -> i64, i64, i64, i64 {
+  ss0 = explicit_slot 8, align=16
+  ss1 = explicit_slot 8, align=16
+  ss2 = explicit_slot 4
+  ss3 = explicit_slot 4
```

Comment:
Do we have a way of enabling an explicit stack check in clif tests? That would be a nice way of verifying that the total stack space required is accounted for.

Reply:
Ah!
```suggestion
function %f0() -> i64, i64, i64, i64 {
  gv0 = vmctx
  ss0 = explicit_slot 8, align=16
  ss1 = explicit_slot 8, align=16
  ss2 = explicit_slot 4
  ss3 = explicit_slot 4
  ss4 = stack_limit gv0
```